docs/modules/releasenotes/pages/whatsnew.adoc

```diff
@@ -2,7 +2,33 @@
 
 = What's New in OpenNMS Horizon 34
 
+
+== System requirements and Dependencies ==
+
+* Java 17: This version of OpenNMS runs on JDKs 17 only.
+* PostgreSQL 10 or higher: This version of Horizon requires any supported version of PostgreSQL from 10 up to (and including) 16.
+* Grafana: This version of OpenNMS is compatible with Grafana _.
+* Elastic Search: If you are using flows, this version will require Elastic Search _.
+
 == New features and important changes
+=== Time To Value Improvements to the Menu Structure ===
+
+One of our 2025-26 goals is to reduce the time it takes a new user, from learning about our product to discovering and visualizing the status of their network infrastructure. Our first step in this approach is the re-organization of the menu structure, with a responsive side menu that streamlines the IT Operator’s actions. Stay tuned for other inprovements in the near futrue such as: OVA Virtual Appliance Support, Adding Windows Management Interface Nodes right from the Quick Add Node, and better Structured Node List navigation.
```

Comment:
Fix a few small typos and also can put each item on its own line (no blank line between them), it will still show up as a single paragraph in the actual documentation but is easier to edit in the `.adoc` file.

```
One of our 2025-26 goals is to reduce the time it takes a new user from learning about our product to discovering and visualizing the status of their network infrastructure.
Our first step in this approach is the re-organization of the menu structure, with a responsive side menu that streamlines the IT Operator’s actions.
Stay tuned for other improvements in the near futuree such as: OVA Virtual Appliance Support, Adding Windows Management Interface Nodes right from the Quick Add Node, and better Structured Node List navigation.
```